### DISCUSSION TITLE
Designate replies in exported log?

### DISCUSSION THREAD

**Discussion by Fallowsthorn** [Tyrrrz/DiscordChatExporter#838] [Q&A] [answered]
Apologies if this is a basic question and/or impossible, my computer skills are pretty much at "print hello world".

A writer friend and I occasionally bring up previous ideas we've had in Discord DMs and elaborate on them using the reply feature. However, when exported, there's no indication of the replied-to message aside from sometimes a yellow background (when the original message is from me), or indeed that the message is a reply at all, so the context gets lost and the conversation looks very disjointed/strange.

Is there a way to indicate the replied-to message, perhaps as a prefix along the lines of "[Reply to [date/time username first 3-5 words of original message]]"? Or even to add a boilerplate "Reply to an earlier message" without any identifying information, if that's all that's possible? I imagine actually linking to the message is specifically a Discord code thing and can't be replicated in HTML or TXT; I'd just like a plaintext marker so I don't forget that a) there was a first part of the conversation and b) I need to go back and integrate it.

**Comment by 96-LB** [comment, marked as answer]
As far as I can tell, the exporter already does this:
![image](https://user-images.githubusercontent.com/21089343/165328382-28462fc4-c419-4bfa-b90d-a10230acaa05.png)


Am I misunderstanding your question?

**Reply by Fallowsthorn** [reply]
...so turns out I had an outdated version, and the auto-updater kept running into some problem or other and not, yknow, updating. Downloading and reinstalling the latest version fixed it and now I can see replies! Extremely quick turnaround on that feature implementation, A+! XD

**Reply by Tyrrrz** [reply]
Considering this was implemented over a year ago, you've probably missed quite a few other features in the meantime ;)
![image](https://user-images.githubusercontent.com/1935960/165939949-81902a55-9d50-4df5-81f1-46c175d550cf.png)
